### PR TITLE
Setting Up A React Environment: Link to prettier and eslint config

### PR DESCRIPTION
**Pull request by nikitarevenco**
## Because

Was confused if there were any extra steps in setting up prettier with eslint. By now, the learner is likely to already know about Prettier and what it is, so pointing to the home page isn't necessary and instead it's better to directly point to the installation instructions. 

Alternatively, we can keep the link to the homepage for Prettier and instead add another sentence linking to instructions.

## This PR

- Remove link pointing to prettier home page
- Add link pointing to instructions for setting up prettier with lint

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: react/introduction/setting_up_a_react_environment.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ All of this may understandably look unlike anything you've seen up until now, bu
 
 ### Keeping it clean
 
-The starter project ships with [ESLint](https://eslint.org/). You could also set up [Prettier](https://prettier.io/) to help keep your React code as clean as can be.
+The starter project ships with [ESLint](https://eslint.org/). You could also set up [Prettier](https://github.com/prettier/eslint-config-prettier) to help keep your React code as clean as can be.
```
Comment (thatblindgeye) [inline react/introduction/setting_up_a_react_environment.md:121]: We actually point to eslint-config-prettier in the JS course > Linting lesson, so the verbiage could be further tweaked:

```suggestion
The starter project ships with [ESLint](https://eslint.org/). If you haven't been doing it or simply forgot about it from our [linting lesson](https://www.theodinproject.com/lessons/node-path-javascript-linting), be sure to also set up [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to help keep your React code as clean as can be.
```

**Review by thatblindgeye — COMMENTED** [review]
Let me know what you think of the below suggestion, but definitely agree we should update that section in some way!

File: react/introduction/setting_up_a_react_environment.md (hunk 1 above)
Comment (nikitarevenco) [reply react/introduction/setting_up_a_react_environment.md]: That's a great idea, don't want to have 2 different places pointing to the same resource!

**Comment by thatblindgeye** [comment]
@nikitarevenco just needs merge conflict resolved otherwise lgtm!

**Comment by nikitarevenco** [comment]
> @nikitarevenco just needs merge conflict resolved otherwise lgtm!

Okay, the merge conflicts have been resolved

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**